testem.js

```diff
@@ -16,6 +19,12 @@ module.exports = {
         "--remote-debugging-port=0",
         "--window-size=1440,900"
       ].filter(Boolean)
+    },
+    Firefox: {
+      ci: [
+        "--headless",
```

Comment:
Did you test this / does this work?

From the documentation the headless flag has only one dash:
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode

Reply:
Interesting, yes I tested it and yes it works. However, you're right I'll change it to match the docs

Reply:
Done

Reply:
Good to know! :-)